migrations/Version202608191410452142_taoDeliveryRdf.php

```
@@ -0,0 +1,43 @@
+        $script = new RegisterSectionVisibilityService();
+        $script->setServiceLocator($this->getServiceLocator());
+        $script([]);
+
+        $this->addReport(
+            Report::createSuccess(
+                'Deliveries tab and Publish action are hidden when FEATURE_FLAG_DISABLE_DELIVERIES is enabled'
+            )
+        );
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)